runtime/src/migrations.rs

```diff
@@ -0,0 +1,162 @@
+pub(crate) mod account_data_migration {
+    use crate::*;
+    use frame_support::log;
+    use pallet_balances::ExtraFlags;
+
+    mod prev {
+        use super::*;
+        use frame_support::{pallet_prelude::ValueQuery, storage_alias, Blake2_128Concat};
+
+        #[derive(
+            Clone, Eq, PartialEq, Default, RuntimeDebug, Encode, Decode, TypeInfo, MaxEncodedLen,
+        )]
+        pub struct AccountDataStruct<Balance> {
+            pub free: Balance,
+            pub reserved: Balance,
+            pub misc_frozen: Balance,
+            pub fee_frozen: Balance,
+        }
+
+        #[derive(
+            Clone, Eq, PartialEq, Default, RuntimeDebug, Encode, Decode, TypeInfo, MaxEncodedLen,
+        )]
+        pub struct AccountStruct<Balance> {
+            pub nonce: u32,
+            pub consumers: u32,
+            pub providers: u32,
+            pub sufficients: u32,
+            pub data: AccountDataStruct<Balance>,
+        }
+
+        #[storage_alias]
+        pub type Account<T: frame_system::pallet::Config> = StorageMap<
+            frame_system::pallet::Pallet<T>,
+            Blake2_128Concat,
+            AccountId,
+            AccountStruct<Balance>,
+            ValueQuery,
+        >;
+    }
+
+    const TARGET: &'static str = "runtime::account_data_migration";
+    pub struct Migration;
```

Comment:
Might wanna give this a more descriptive name.

Reply:
I mean it is the only Migration in the `account_data_migration` module :P